.circleci/config.yml

```diff
@@ -0,0 +1,267 @@
+version: 2.0
+
+aliases:
+  # Workspace
+  - &workspace
+    ~/video-quickstart-android
+
+  - &gradle-cache-key
+    key: jars-{{ checksum "build.gradle" }}-{{ checksum "gradle/wrapper/gradle-wrapper.properties" }}
+  - &restore-cache-gradle
+    <<: *gradle-cache-key
+    name: Restore Gradle Cache
+  - &save-cache-gradle
+    <<: *gradle-cache-key
+    name: Save Gradle Cache
+    paths:
+      - ~/.gradle/caches
+      - ~/.gradle/wrapper
+
+  # Containers
+  - &build-defaults
+    working_directory: *workspace
+    docker:
+      - image: circleci/android:api-28-node8-alpha
+    environment:
+      - _JAVA_OPTIONS: "-XX:+UnlockExperimentalVMOptions -XX:+UseCGroupMemoryLimitForHeap"
+
+jobs:
+  setup-workspace:
+    <<: *build-defaults
+    resource_class: medium+
+    steps:
+      # Setup code and workspace for downstream jobs
+      - checkout
+      - restore-cache: *restore-cache-gradle
+      
+      # Save cache
+      - save-cache: *save-cache-gradle
+
+  build-exampleAdvancedCameraCapturer:
+    <<: *build-defaults
+    resource_class: large
+    steps:
+      # Setup
+      - checkout
+      - attach_workspace:
+          at: *workspace
+      - restore-cache: *restore-cache-gradle
+
+      # Build app
+      - run:
+          name: Build exampleAdvancedCameraCapturer
+          command: ./gradlew -q exampleAdvancedCameraCapturer:assemble
+
+      # Save the builds to workspace
+      - persist_to_workspace:
```

Comment:
We are not using workspace feature this workflow so let's remove all the `persist_to_workspace` from the jobs.